psi4/driver/schema_wrapper.py

```diff
@@ -178,17 +201,63 @@ def _clean_psi_environ(do_clean):
         psi4.core.clean_options()
         psi4.core.clean()
 
+
 def _read_output(outfile):
     try:
         with open(outfile, 'r') as f:
             output = f.read()
 
         return output
     except OSError:
-        return "Could not read output file"
+        return "Could not read output file."
+
+
+def _quiet_remove(filename):
+    """
+    Destroy the created at file at exit, pass if error.
+    """
+    try:
+        os.unlink(filename)
+    except OSError:
+        pass
+
+
+def run_qcschema(input_data, clean=True):
+
+    outfile = os.path.join(core.IOManager.shared_object().get_default_path(), str(uuid.uuid4()) + ".qcschema_tmpout")
+    core.set_output_file(outfile, False)
+
+    try:
+        input_model = qcng.util.model_wrapper(input_data, qcel.models.ResultInput)
+
+        # qcschema should be copied
+        ret_data = run_json_qcschema(input_model.dict(encoding="json"), clean, False)
```

Comment:
why encoding json? won't that be lossy?

Reply:
Looks like the `from_dict` can take array so I can drop this.

Reply:
ok. fields `from_dict` is expecting aren't the fields in `models.Molecule`. maybe i'm not following.

Reply:
`psi4.core.Molecule.from_dict` can take in NumPy arrays, not just lists. This wasn't clear to me before.